Updates the SSO docs to also cater to classic/Oauth based flow. (#9251)
circleci/circleci-docs: jekyll/_cci2/authentication/sign-in-to-an-sso-enabled-organization.adoc

[join-an-organization]
== Join an organization

To join a CircleCI organization that has SSO enabled you must first have an active CircleCI account that you log into with an email and password or with social logins like GitHub or Bitbucket. For more information on creating a CircleCI account, see the xref:../first-steps#[Sign up and try] page.

> NOTE: For users of an organisation that are using OAuth/social login to join CircleCI, it is important to SSO authorize in the VCS as well as in CircleCI.

Author: I could link the [github](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github) and [bb](https://support.atlassian.com/atlassian-account/docs/log-in-with-saml-single-sign-on/) authentication pages

> NOTE: For users of an organisation that are using OAuth/social login to join CircleCI, it is important to SSO authorize in the VCS as well as in CircleCI.

Reviewer: ```suggestion
NOTE: If you use a OAuth/social login to join CircleCI, it is important to SSO authorize in GitHub/Bitbucket as well as in CircleCI.
```
